devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/java/build.gradle-template.ftl

```diff
@@ -38,5 +38,9 @@ compileJava {
     options.compilerArgs << '-parameters'
 }
 
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
```

Comment:
You can also remove the `java {}` and declare those two properties in the root (not sure how Gradle users do tbh)

Reply:
I think what you  describe is the old way of doing things, although I am not sure :)